tests/end2endtest/utils/pytest_extensions.py

```diff
@@ -88,10 +79,10 @@ def server_from_docker(image_name, image_port, host_port, container_name,
         try:
             inspect_out = subprocess.check_output(
                 ['docker', 'container', 'inspect', container_name],
-                stderr=DEVNULL)
-        except EnvironmentError as exc:
+                stderr=subprocess.DEVNULL)
+        except OSError as exc:
```

Comment:
Another exception change. Is this correct?

Reply:
Yes, it is correct. See https://docs.python.org/3/library/exceptions.html#OSError.filename2